proposals/2018-10-25-selecting-between-python-2-and-3.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Selecting Between Python 2 and 3
+status: Draft
+created: 2018-10-25
+updated: 2018-11-02
+authors:
+  - [brandjon@](https://github.com/brandjon)
+reviewers:
+  - TBD
+discussion thread: [bazel #6583](https://github.com/bazelbuild/bazel/issues/6583)
+---
+
+# Selecting Between Python 2 and 3
+
+## Abstract
+
+The "Python mode" configuration value controls whether Python 2 or Python 3 is used to run Python targets built by Bazel. This doc reviews the existing mechanisms for setting the Python mode (the "tri-state model") and describes a simplified mechanism that should replace it (the "boolean model").
```

Comment:
nit: doc -> documentation (in the interests of I18N)

Reply:
-> document

Reply:
Done.